src/main/java/com/github/bbooong/bangumall/auth/application/dto/LoginRequest.java

```diff
@@ -0,0 +1,3 @@
+package com.github.bbooong.bangumall.auth.application.dto;
+
+public record LoginRequest(String email, String password) {}
```

Comment:
앞에 도메인 이름을 넣어서 카테고리를 짓는 건 어떤가요?

```suggestion
public record AuthLoginRequest(String email, String password) {}
```